src/check-attributes-buffer-overflow.js

```diff
@@ -20,6 +21,11 @@ function getLastUsedIndexForDrawElements(gl, funcName, startOffset, vertCount, i
     errors.push('No ELEMENT_ARRAY_BUFFER bound');
     return undefined;
   }
+  const eboType = getEBOTypeByVertCount(vertCount);
```

Comment:
AFAICT, This is not a valid check. It's perfectly valid to have a vertCount more than the max index size.

Reply:
https://jsgist.org/?src=c2482170915d3a2039dacd84bf1fd6b2

Reply:
ok,that's not a valid,just a tips for the data type